dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/EnhancedResultsFigureGenerator.java

```diff
@@ -511,19 +584,23 @@ else if (techniqueName.equals("prioritization")) {
                             if (orderName.equals("absolute")) {
                                 // T3
                                 fig17_array[0] = apfd_value;
+                                currProj.setNumTotalDependentTests(17, 0, numTotal);
```

Comment:
You should add comments in this file and possibly in ProjectEnhancedResults explaining what each of these magical numbers represent. Ex. 17 represents the enhanced prioritization results figure in the paper since it is figure 17 in the paper. Also 0 represents the T3 because T3's result is in column 0.